app/org/sagebionetworks/bridge/util/DuplicateZipEntryException.java

```diff
@@ -0,0 +1,13 @@
+package org.sagebionetworks.bridge.util;
```

Comment:
These will still be returned as 500s, but are they subclasses of BridgeServiceException?